backend/app/adapter/graphql/resolver/url_test.go

```diff
@@ -0,0 +1,44 @@
+package resolver
+
+import (
+	"github.com/byliuyang/app/mdtest"
+	"short/app/adapter/graphql/scalar"
+	"short/app/entity"
+	"testing"
+	"time"
+)
+
+func TestURL_Alias(t *testing.T) {
+	urlResolver := URL{url: entity.URL{Alias: "TestAlias"}}
+
+	expected := urlResolver.url.Alias
+	got := *urlResolver.Alias()
+	mdtest.Equal(t, got, expected, "*urlTest.Alias() = %v; want %v", expected, got)
+}
+
+func TestURL_OriginalURL(t *testing.T) {
+	urlResolver := URL{url: entity.URL{OriginalURL: "TestOriginalUrl"}}
+
+	expected := urlResolver.url.OriginalURL
+	got := *urlResolver.OriginalURL()
+	mdtest.Equal(t, got, expected, "*urlResolver.OriginalURL() = %v; want %v", expected, got)
+}
+
+func TestURL_ExpireAt(t *testing.T) {
+	timeAfter := time.Now().Add(5 * time.Second)
+	testCases := []struct {
+		Url    URL
```

Comment:
```suggestion
		url    URL
```
We are exporting Url out of this package.